BUILD.md

```diff
@@ -5,7 +5,7 @@
     - On MacOS these may already be available if you have xcode libraries installed
 2. In the Cedille repository directory:
     - Run `stack build Agda alex happy`
-    - Run `stack build --copy-bins --local-bin-path .`
+    - Run `mkdir -p ./bin; stack build --copy-bins --local-bin-path ./bin`
```

Comment:
Alternatively you could ship this repo with `./bin/empty` in there to make the `mkdir` redundant.